server/service/live_queries.go

```diff
@@ -292,6 +288,13 @@ func (svc *Service) RunLiveQueryDeadline(
 				}
 			}()
 
+			readChan, cancelFunc, err := svc.GetCampaignReader(ctx, campaign)
+			if err != nil {
+				resultsCh <- fleet.QueryCampaignResult{QueryID: queryID, Error: ptr.String(err.Error()), Err: err}
```

Comment:
While we're at it let's add a level.Error here too.

Reply:
Done